.github/workflows/My-ci-pipeline.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
+      - '.github\**'
+      - '.idea\**'     
+      - '.pytest_cache\**'
+      - '.venv\**'
+      - '.vscode\**'
+
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '**.lock'
+      - '.github\**'
+      - '.idea\**'     
+      - '.pytest_cache\**'
+      - '.venv\**'
+      - '.vscode\**'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: BuildTestImage
+      run: docker build --target test --tag todo-app:test .
+    
+    - name: BuildTestImage
```

Comment:
We could probably choose a better name for this step!

Reply:
Changed to BuildAndTestImage